drivers/modem/modem_cellular.c

```diff
@@ -872,11 +885,13 @@ static void modem_cellular_init_power_off_event_handler(struct modem_cellular_da
 
 	switch (evt) {
 	case MODEM_CELLULAR_EVENT_TIMEOUT:
-		if (modem_cellular_gpio_is_enabled(&config->power_gpio)) {
+		if (IS_ENABLED(CONFIG_MODEM_CELLULAR_USE_POWER_DOWN_SCRIPT) && (config->power_down_script != NULL)) {
```

Comment:
I think just checking for power down script is enough here

Reply:
I am not sure I want to have a `CONFIG_MODEM_CELLULAR_USE_POWER_DOWN_SCRIPT` in `steelix-app` that won't exist on every other branch of `zephyr`